src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -947,8 +947,10 @@ components:
           example: c88a6c48-97e2-4c04-93b5-98619966447b
         source:
           $ref: '#/components/schemas/Source'
-        conditionType:
-          $ref: '#/components/schemas/ReferenceData'
```

Comment:
I think the `ReferenceData` type should only be used/referenced in the reference data endpoints, otherwise we are possibly mixing concerns a little?